code/modules/clothing/under/jobs/security.dm

```diff
@@ -37,6 +37,16 @@
 	siemens_coefficient = 0.8
 	armor_list = list(melee = 5, bullet = 5, energy = 5, bomb = 0, bio = 0, rad = 0)
 
+/obj/item/clothing/under/rank/security/cadet
```

Comment:
These two should be merged into 1 type that can be swapped

Reply:
suuuuuuure.